ivy/functional/frontends/tensorflow/linalg.py

```diff
@@ -43,3 +43,7 @@ def eye(num_rows, num_columns=None, batch_shape=None, dtype=ivy.float32, name=No
 
 
 eye.unsupported_dtypes = {"torch": ("float16", "bfloat16")}
+
+
+def band_part(input, num_lower, num_upper, name=Nome):
+    return ivy.band_part(input, num_lower, num_upper)
```

Comment:
I don't think there's a `band_part` function in Ivy's functional API, so either you'll have to implement the `tensorflow` `band_part` frontend as a composition of existing Ivy functions or extend Ivy's functional API by adding `ivy.band_part` in a separate PR referring to the [docs](https://lets-unify.ai/ivy/contributing/4_open_tasks.html#ivy-api-extensions)